src/components/common/Actions.vue

```diff
@@ -219,7 +218,6 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
         !this.isValidDateTime(this.getEffectiveDateTime.effectiveDate)
       ) {
         this.setEffectiveDateTimeValid(false)
-        this.setIsFilingPaying(false)
```

Comment:
This is done again on line 224

Reply:
Please add comment in code here and line 200 why we don't want to clear the flags.

Reply:
Will do!